app/jobs/assignment_repo/create_github_repository_job.rb

```diff
@@ -38,10 +38,9 @@ def perform(assignment, user, retries: 0)
           message: IMPORT_STARTER_CODE,
           user: user,
           invite_status: invite_status,
-          percent: user.feature_enabled?(:repository_import_webhook) ? 50 : 0,
+          percent: 50,
```

Comment:
Why is this hard coded as `50`?

Reply:
We should actually get rid of the `percent` field entirely. These percentages are no longer meaningful.